src/networks/mainnet/mod.rs

```diff
@@ -23,9 +25,17 @@ pub static GENESIS_CID: Lazy<Cid> = Lazy::new(|| {
     Cid::from_str("bafy2bzacecnamqgqmifpluoeldx7zzglxcljo6oja4vrmtj7432rphldpdmm2").unwrap()
 });
 
-/// Default bootstrap peer ids.
-pub const DEFAULT_BOOTSTRAP: &[&str] =
-    &const_str::split!(include_str!("../../../build/bootstrap/mainnet"), "\n");
+pub static DEFAULT_BOOTSTRAP: Lazy<Vec<Multiaddr>> = Lazy::new(|| {
+    let default_bootstrap = include_str!("../../../build/bootstrap/mainnet")
```

Comment:
The code in the lambda is a bit repetitive, may be worth making it a separate function, for upcoming networks

Reply:
I like it.